kitsune/sumo/static/sumo/js/form-wizard.js

```diff
@@ -34,14 +35,23 @@ export class FormWizard extends HTMLElement {
   static get markup() {
     return `
       <template>
-        <div class="form-wizard-content">
-          <h2>${gettext("Wizard header")}</h2>
-          <section>
-            <ul id="step-indicator"></ul>
-            <slot name="active"></slot>
-          </section>
+        <div class="form-wizard-root">
+          <div class="form-wizard-content">
+            <h2>${gettext("Wizard header")}</h2>
+            <section>
+              <ul id="step-indicator"></ul>
+              <slot name="active"></slot>
+            </section>
+          </div>
+          <progress max="100" value="0"></progress>
+          <div class="form-wizard-disqualified">
+            <div class="disqualification">
+              <img class="warning-icon" src="${warningImageURL}"/>
```

Comment:
```suggestion
              <img class="warning-icon" src="${warningImageURL}" aria-hidden="true" />
```

Let's keep this icon thing going